src/main/java/ATM.java

```diff
@@ -0,0 +1,25 @@
+package src.main.java;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ATM {
+	Map<String,ATMaccount> accounts = new HashMap<>();
+	private Application app;
+	public ATM(Application app) {
+		this.app = app;
+	}
+	public ATMaccount getAccountUser(String name) {
+		if(!accounts.containsKey(name)) {
+			ATMaccount account = app.makeAccount(name,this);
```

Comment:
From version 10, Java has a thing called [local variable type inference](https://developer.oracle.com/learn/technical-articles/jdk-10-local-variable-type-inference). You could replace every occurrence of a variable declaration in a method without decalring it's type. Start from the `account` variable right here.

Reply:
I'll do that

Reply:
is it actually that important? I'm asking because I don't see why it would be a problem?

Reply:
It's important to form the correct habits from the start, in general. So, it's a good idea to use type inference since it's a feature introduced in Java to help with the varbosity and to make Java a more modern language. Languages like Kotlin, even though they are still strongly typed languages, have this feature enabled by default.

Reply:
ok I see thx